tests/integration/test_app_integration.py

```
@@ -0,0 +1,165 @@
+        self.app_context = test_app.app_context()
+        self.app_context.push()
+
+        db.create_all()
+        self.db = db
+
+    def tearDown(self):
+        """Clean up"""
+        self.db.session.remove()
+        try:
+            self.db.drop_all()
+        except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)